### ISSUE TITLE
Code of Conduct

### ISSUE THREAD

**Issue by xamebax** [euruject/euruject#5]
Just sayin'.

I guess I can be responsible for drafting one, but not before June.

**Comment by plexus** [comment]
Good point! I'm guessing we can stick close to the eurucamp CoC. June is fine.

**Comment by xamebax** [comment]
Oh, that would be easiest! One issue I'm foreseeing: in the case of eurucamp, it's quite simple to know who to contact, because the conference staff has t-shirts. Who should be a contact point here? I think it would be cool to have a couple volunteers named before the event. I think it's better than having to contact random people. We don't have to make decisions now though.

**Comment by plexus** [comment]
Again, good point! I guess this is where the "all attendees are organizers" thing kind of breaks down.

I suppose we will need to establish some kind of "core group", since in the case of unwanted behavior we also need to be able to kick people out. We can publish, and signpost, names and phone numbers. I agree we can postpone this until things in general become a bit more concrete.

I also saw this some time back (too me some time to find it again!), and I quite like it

https://twitter.com/scalzi/status/462332509582155776/photo/1

**Comment by skade** [comment]
There will always be some staff handling the place. I recommend to brief all those using some kind of handling procedure. I like the one from pycon:

https://us.pycon.org/2013/about/code-of-conduct/harassment-incidents-staff/

Also, last years eurucamp staff just had badges and no special shirts. We still have quite a few of them lying around and can give you some (minimum batch size for buying was 100).

**Comment by xamebax** [comment]
Personally, I think we could just sign this CoC, since ♥ ♥ ♥ ♥ RUG-B ♥ ♥ ♥ ♥ did a wonderful job: http://rubyberlin.github.io/code-of-conduct/

@skade, I think badges are a good idea, great. :)

**Comment by skade** [comment]
You can also just adopt the eurucamp CoC ;). The PDX CoC was created for user groups.

**Comment by plexus** [comment]
I had another look at the eurucamp and the Ruby Berlin CoC. The latter seems to be suitable for both user groups and conferences, and we fall a bit in the middle anyway, so I went with that one. The eurucamp also has some things that don't apply to us, e.g. we don't have sponsors.

We still need to list a few contact people. I'm fine with being on there. @xamebax would you like to be on there as well? Anyone else who comes to mind, maybe someone a bit more outside of our direct circle?

**Comment by xamebax** [comment]
Thanks for refreshing this topic, I kinda forgot about it, ups. Yeah, I'd love to be on the list. Should even have a German phone number by then, for my one-year-in-Berlin anniversary.

**Comment by xamebax** [comment]
>  Anyone else who comes to mind, maybe someone a bit more outside of our direct circle?

I tweeted about it last night, but to be honest, I think that if you don't ask directly, people probably won't respond. Which makes the whole "outside of our direct circle" thing more difficult. Maybe we should put a list of people we need in the readme? That's always a couple clicks less for someone.

**Comment by plexus** [comment]
From @nerdbabe's reply I'm assuming she's ok with being a contact person :) With you, me, and @bitboxer as "organizers" I think that's enough coverage for such a small event. The important part is that it's clear where people can turn to, and that we all know how to act in case of incident. I will review the handling procedure @skade posted, we can then reiterate that in the morning.

It's a good idea to explicitly list the roles people can play at Euruject. What comes to mind:
- help setting up (be there at 9:00)
- help clean up (stay after 20:00)
- give a workshop
- give a lightning talk
- give a live demo

I suppose we also want some people to welcome those that arrive and explain the concept a bit, and perhaps a few hosts to moderate the open stage. I'm happy to do that but preferably not the whole day.

**Comment by xamebax** [comment]
> From @nerdbabe's reply I'm assuming she's ok with being a contact person :)

It was late and I admit I didn't understand the reply. I'm sorry. Awesome, thanks @nerdbabe!

**Comment by plexus** [comment]
Maybe I'm assuming too much though, I'll take it up in private.

**Comment by nanoparsec** [comment]
I am in with helping. :) And I think my SoC pair Magda will help as well.

**Comment by PragTob** [comment]
Y U NO PR ON berlin code of conduct repo to add yourself as a supporter?!?!?!?! https://github.com/rubyberlin/code-of-conduct

(if you want to you may list me as a coc contact person)

**Comment by xamebax** [comment]
> Y U NO PR ON berlin code of conduct repo to add yourself as a supporter?!?!?!?!

I should have time to deal with that today.
